docs/rules/no-duplicate-imports.md

```diff
@@ -2,6 +2,8 @@
 
 An ES6/ES2015 import can be spread over multiple lines, but this takes up unneeded whitespace. This rules validates that all imports from a single module exists in a single import statement.
 
+Using a single import statement per module will make code much clearer; you can see everything being imported from that module on one line, rather than it being spread out over several lines.
```

Comment:
`import` statement

s/much clearer; you/clearer because you/

Because several lines is in previous paragraph, I think you can stop at: on one line.